discretize/utils/mesh_utils.py

```diff
@@ -712,6 +741,17 @@ def refine_tree_xyz(
             mesh.finalize()
 
     elif method.lower() == "surface":
+        warnings.warn(
+            "The surface option is deprecated as of `0.9.0` please update your code to "
+            "use the `TreeMesh.refine_surface` functionality. It will be removed in a "
+            "future version of discretize.",
+            DeprecationWarning,
+        )
+        # padding = np.zeros((len(octree_levels), mesh.dim))
+        # padding[:, -1] = np.maximum(octree_levels - 1, 0)
+        # padding[:, :-1] = octree_levels_padding[:, None]
+        # padding = padding[non_zeros]
+        # mesh.refine_surface(xyz, levels, padding, finalize=finalize, pad_down=True, pad_up=False)
```

Comment:
Why not re-routing to the new method now? These lines don't run as is BTW

Reply:
These are here to mostly just remind ourselves of the translations between the two methods, as their calling conventions are slightly different, and the surface refine behaves slightly differently around the edges.